docker/src/main/java/brooklyn/entity/container/docker/application/VanillaDockerApplication.java

```diff
@@ -54,6 +54,9 @@
     @SetFromFlag("imageTag")
     ConfigKey<String> IMAGE_TAG = DockerAttributes.DOCKER_IMAGE_TAG.getConfigKey();
 
+    @SetFromFlag("entrypoint")
+    ConfigKey<List<String>> IMAGE_ENTRYPOINT = DockerAttributes.DOCKER_IMAGE_ENTRYPOINT.getConfigKey();
```

Comment:
would it make sense to rename the other config in this file as well then, so that everything matches there corresponding variables in DockerAttributes?